src/Network-Tests/UUIDGeneratorTest.class.st

```diff
@@ -18,67 +18,75 @@ UUIDGeneratorTest >> setUp [
 
 { #category : #testing }
 UUIDGeneratorTest >> testCounterRollover [
+
 	| previousUuid uuid |
-	previousUuid := generator next.
+	previousUuid := ByteArray new: 16.
+	UUIDGenerator default placeFields: previousUuid.
 	"the internal counter inside generator will rollover at 16r10000,
 	the least significant 10th byte rolls over every 16r100,
 	the internal counter starts at a random value"
 	2 * 16r10000 timesRepeat: [
-		uuid := generator next.
+		uuid := ByteArray new: 16.
+		UUIDGenerator default placeFields: uuid.
 		"same generator, consequetive counters (take roll-over into account)"
-		self assert: ((previousUuid at: 10) + 1) \\ 256 equals: (uuid at: 10).
+		self assert: (previousUuid at: 10) + 1 \\ 256 equals: (uuid at: 10).
 		previousUuid := uuid ]
 ]
 
 { #category : #testing }
 UUIDGeneratorTest >> testDefault [
-	self assert: UUIDGenerator default equals: UUIDGenerator default.
-	self deny: UUIDGenerator next isNilUUID
-]
 
-{ #category : #testing }
-UUIDGeneratorTest >> testOne [
-	| uuid |
-	uuid := generator next.
-	self assert: (uuid isKindOf: UUID).
-	self deny: uuid isNilUUID
+	self assert: UUIDGenerator default equals: UUIDGenerator default.
 ]
 
 { #category : #testing }
 UUIDGeneratorTest >> testSpeed [
+	|array|
+	array := ByteArray new: 16.
 	self
-		should: [ 100000 timesRepeat: [ generator next ] ]
+		should: [ 100000 timesRepeat: [ UUIDGenerator default placeFields: array ] ]
```

Comment:
I don't like this test.

First, it's breaking the public API. Why testing `placeFields:` and not directly `UUID new`.
Then, testing speed like this is just a smoke test. Why 100000 and not more (or less?) :D.
I'd remove this test altogether.